driver/main.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
You have to update bblfsh-sdk and run `bblfsh-sdk update` again, since we updated `main.go` again. Impact should be minimal.